utils/query.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func ParseQueryResponse(response *ceresdbproto.QueryResponse) ([]map[string]interface{}, error) {
+	// schema is nil in sql CREATE Table or DROP Table
```

Comment:
schema is nil when SQL is DDL(such as create/drop table)

Reply:
done